examples/k8s.yaml

```diff
@@ -115,7 +115,7 @@ data:
       batch:
       memory_limiter:
         # Same as --mem-ballast-size-mib CLI argument
-        ballast_size_mib: 165
+        ballast_size_mib: 683
         # 80% of maximum memory up to 2G
         limit_mib: 400
```

Comment:
`limit_mib` seems to be very low for 2G memory limit, same for `spike_limit_mib`. looks like copied over from the agent config
Should we adjust them as well?

Reply:
ah, will fix.